plinth/modules/letsencrypt/templates/letsencrypt.html

```diff
@@ -101,5 +101,7 @@
   </div>
 
   {% include "diagnostics_button.html" with module="letsencrypt" %}
-
+  {% else %}
+  You do not have any domain configured. Configure domains to be able to obtain certificates for them.
+  {% endif %}
```

Comment:
* Text here needs to be indented.
* Text needs to be internationalized so the translators can translated into various languages.
* Slightly passive tone "No domains have been configured. ..." would be nice.